Add additional information about Neovim

mikkovedru opened the pull request.

Kethku commented:
This makes sense to add. Could you change your PR to target the opengl branch though?

mikkovedru commented:
@Kethku 
Done. Hopefully, I did everything correctly as this is my first time using github's graphical interface to fix merge conflicts.

Kethku commented:
Hmmm. Weird that it included a bunch of other changes... I'm just gonna cherry pick your change instead. Sorry for the confusion.

Kethku commented:
Thanks for the PR!

j4qfrost commented on src/bridge/mod.rs:
```diff
@@ -35,7 +35,11 @@ fn set_windows_creation_flags(cmd: &mut Command) {
     if SETTINGS.get::<CmdLineSettings>().wsl {
         let mut cmd = Command::new("wsl");
-        cmd.arg(bin);
+        cmd.args(&[
```
What is this change for?

Kethku replied:
I think this was a merge error

j4qfrost replied:
It's quite curious. That command looks like it would set PATH to an empty string. Was curious if wsl had some special properties.

j4qfrost replied:
Aaa nvm, it prints error to null.

j4qfrost commented on src/bridge/events.rs:
```diff
@@ -117,7 +117,7 @@ pub enum WindowAnchor {
 }
 
-#[derive(Clone, Debug)]
+#[derive(Debug, Clone)]
```
Was this suggested by fmt?

Kethku replied:
I think this was a merge error

mikkovedru commented:
> Hmmm. Weird that it included a bunch of other changes... I'm just gonna cherry pick your change instead. Sorry for the confusion.

Yeah, I don't know what happened... I edited the PR, changed the target from master branch to opengl branch and a bunch of other commits showed up (only one of which actually touched the same readme file I modified).

Glad to help! :-)
